apps/src/javalab/JavalabConsole.jsx

```diff
@@ -130,7 +135,13 @@ class JavalabConsole extends React.Component {
         <PaneHeader hasFocus={true}>
           <PaneSection>Console</PaneSection>
         </PaneHeader>
-        <div style={style.consoleStyle} ref={el => (this._consoleLogs = el)}>
+        <div
+          style={{
+            ...style.consoleStyle,
+            ...(this.props.isDarkMode ? style.darkMode : style.lightMode)
```

Comment:
Given the tight visual coupling between the console and editor, losing the ability to change between dark and light in the console in the future would be confusing to the user. Adding a unit test here would help prevent that.